crates/zksync/core/src/lib.rs

```diff
@@ -74,6 +74,15 @@ pub fn get_nonce_key(address: Address) -> rU256 {
     zksync_types::get_nonce_key(&address.to_h160()).key().to_ru256()
 }
 
+/// Represents additional data for ZK transactions that require a paymaster.
+#[derive(Clone, Debug, Default)]
+pub struct ZkPaymasterData {
```

Comment:
I wonder why we use `PaymasterData` here and `PaymasterParams` in other places while the fields are pretty much the same. Any reason to make the distinction between data and params?

Reply:
Makes sense I would use params for both in order to be consistent but that's probably personal preference.